/**:
  ros__parameters:
    pointcloud2_filter_chain:
      filter1:
        name: voxel_filter
        type: filters_pointcloud2_erdc/VoxelGridFilter
        params:
          x_size: 1.0
          y_size: 1.0
          z_size: 0.2
